voorhoede/voorhoede-website
# feat/blog-post-scripts

## What changes were made

- enable scripts from the cms to run in blog posts
- added custom-script component
- added noscript warning to help users with javascript disabled understand that the page may not function as explained in the blog post

## Ticket

https://trello.com/c/SJizgBYO/753-as-a-writer-i-want-to-add-javascript-to-my-blog-post-so-i-can-have-interactive-examples

## How to test or check results

- check deploy preview: in my branch, the 'scroll-driven-animations-for-better-transitions' blog post (in 'en' locale) has a script that will log something to the console and shows a noscript if JS is disabled

## Checks
